README.md

```diff
@@ -7,4 +7,48 @@
 
 ## Getting started
 
-See [Contributing](./CONTRIBUTING.md) for installation.
\ No newline at end of file
+See [Contributing](./CONTRIBUTING.md) for installation.
+
+
+## Localization with variable 
+
+#### 1. Add text with variable in json ####
```

Comment:
WHY USING `####` INSTEAD OF `###`? There isn't any `###` parent so you may use it.